code/modules/admin/view_variables/topic.dm

```diff
@@ -316,6 +316,53 @@
 			return
 		holder.Topic(href, list("makeai"=href_list["makeai"]))
 
+	else if(href_list["addailment"])
+
+		var/mob/living/carbon/human/H = locate(href_list["addailment"])
+		if(!istype(H))
+			to_chat(usr, "This can only be done to instances of type /mob/living/carbon/human")
+			return
+		var/obj/item/organ/O = input("Select a limb to add the ailment to.", "Add Ailment") as null|anything in (H.organs|H.internal_organs)
+		if(!O)
+			return
+		var/list/possible_ailments = list()
+		for(var/atype in subtypesof(/datum/ailment))
+			var/datum/ailment/ailment = get_ailment_reference(atype)
+			if(ailment && ailment.category != ailment.type && ailment.can_apply_to(O))
+				possible_ailments |= ailment
+
+		var/datum/ailment/ailment = input("Select an ailment type to add.", "Add Ailment") as null|anything in possible_ailments
+		if(!istype(ailment))
+			return
+		if(!QDELETED(O) && O.add_ailment(ailment))
```

Comment:
Need to do another recheck that it's a valid attached limb, and also maybe the previous valid ailment check if that's contingent on limb status.